src/viam/sdk/services/mlmodel/client.cpp

```diff
@@ -23,10 +23,7 @@ namespace sdk {
 
 namespace {
 
-struct tensor_storage_and_views {
-    mlmodel_details::tensor_storage storage;
-    MLModelService::named_tensor_views views;
-};
+constexpr bool kUseFlatTensors = true;
```

Comment:
I've opted to leave the existing struct based implementation in place so I can easily A/B test. All code selected by `kUseFlatTensors = false` would be removable if we decided to make a breaking change.